node/narwhal/examples/simple_node.rs

```diff
@@ -398,6 +399,35 @@ async fn start_server(bft: Option<BFT<CurrentNetwork>>, primary: Primary<Current
 
 /**************************************************************************************************/
 
+/// The operating mode of the node.
+#[derive(Debug, Clone, ValueEnum)]
+enum Mode {
+    /// Runs the node with the Narwhal memory pool protocol.
+    Narwhal,
+    /// Runs the node with the Bullshark BFT protocol (on top of Narwhal).
+    Bft,
+}
+
+/// A simple CLI for the node.
+#[derive(Parser, Debug)]
+struct Args {
+    /// The mode to run the node in.
+    #[arg(long)]
+    mode: Mode,
+    /// The ID of the node.
+    #[arg(long, value_name = "ID")]
+    node_id: u16,
```

Comment:
nit: I'd prefer this be `id`

Reply:
I renamed this in the CLI but didn't touch the usages of `node_id` that predate the PR, happy to adjust these as well too if that is preferred.